components/kyma-environment-broker/internal/storage/postsql/read.go

```diff
@@ -685,6 +685,19 @@ func (r readSession) ListInstances(filter dbmodel.InstanceFilter) ([]dbmodel.Ins
 		nil
 }
 
+func (r readSession) ListEvents(filter dbmodel.EventFilter) ([]dbmodel.EventDTO, error) {
+	var events []dbmodel.EventDTO
+	stmt := r.session.Select("*").From("events")
+	if filter.InstanceID != "" {
+		stmt.Where(dbr.Eq("instance_id", filter.InstanceID))
+	}
+	if filter.OperationID != "" {
+		stmt.Where(dbr.Eq("operation_id", filter.OperationID))
+	}
+	_, err := stmt.Load(&events)
```

Comment:
did you think about adding `order by` part to the SQL statement? usually we expects ordered events (by created_at)

Reply:
good point, I will add that